src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -246,19 +246,28 @@ public class VmrUpdater : VmrManagerBase, IVmrUpdater
 
         reposToUpdate.Enqueue(new DependencyUpdate(mapping, targetRevision, targetVersion, null));
 
+        string commitMessage = $"Recursive update for {mapping.Name} / {GetCurrentVersion(mapping)} → {targetRevision}";
```

Comment:
Nit: Might be over-cautious, but since people might be using different tools and pagers when browsing commit messages, I'd avoid using non-ascii characters and replace '→' with "->". It would also make grepping etc. easier. The same applies to logging, non-standard characters are difficult to grep.

Reply:
I was thinking about this but git commit messages support UTF-8 and the arrow did work well over SSH, on Mac, on Windows, WSL, in combination of these, on GitHub, in my git UI clients too. So I just went with it and it actually makes things much more readable.